loopbackStarter/manifest.yml

```diff
@@ -0,0 +1,10 @@
+applications:
+- path: .
+  memory: 1G
+  instances: 1
+  domain: mybluemix.net
+  name: loopback-starter-app
+  host: loopback-starter-app
+  disk_quota: 1G
+env:
+  NODE_ENV: bluemix
```

Comment:
since the application doesn't work without a service, the manifest.yml should include a `services` section:
```
  services:
  - loopback-mongodb-starter
```